diff --git a/addons/product/product.py b/addons/product/product.py
index c599ce7..81c9748 100644
--- a/addons/product/product.py
+++ b/addons/product/product.py
@@ -531,7 +531,7 @@ class product_template(osv.osv):
         'standard_price': fields.property(type = 'float', digits_compute=dp.get_precision('Product Price'), 
                                           help="Cost price of the product template used for standard stock valuation in accounting and used as a base price on purchase orders. "
                                                "Expressed in the default unit of measure of the product.",
-                                          groups="base.group_user", string="Cost Price"),
+                                          groups="base.group_user,base.group_portal", string="Cost Price"),
         'volume': fields.float('Volume', help="The volume in m3."),
         'weight': fields.float('Gross Weight', digits_compute=dp.get_precision('Stock Weight'), help="The gross weight in Kg."),
         'weight_net': fields.float('Net Weight', digits_compute=dp.get_precision('Stock Weight'), help="The net weight in Kg."),
